### PR TITLE
Add multiple attribute to support multiselect

### PR DESCRIPTION
**Pull request by tomalec**
Default is single select, so should be backward compatible

### CODE REVIEW

**Comment by warpech** [comment]
It replaces the current API completely. In our projects we are not using the current API anywhere, right?

**Comment by warpech** [comment]
There never was a single release of this repo, so I guess it's not any problem actually.

File: juicy-dom-tree-view.html [hunk 1]
```diff
@@ -107,6 +170,10 @@
                 li.className = 't' + node.childNodes[i].nodeType;
                 // add support for shady DOM elements
                 li.className += ' juicy-dom-tree-view';
+                // re-select clones for previously selected elements
+                if(selectedElements && selectedElements.indexOf(li._juicyDocumentElement) > -1 ){
```
Comment (warpech) [inline juicy-dom-tree-view.html:174]: ~~`selectedElements` is always defined, so the first check is redundant.~~

Comment (warpech) [reply juicy-dom-tree-view.html:174]: Sorry, I was not CTRL+F'ing in the complete file... Disregard my comment.

**Review by warpech — APPROVED** [review]
I can see that you tried to do the minimum amount of changes. LGTM!

**Comment by tomalec** [comment]
> It replaces the current API completely.

not complately, it just removes `selectedClones`

>  In our projects we are not using the current API anywhere, right?

In the layout editor. So, the question is are we going to support the previous (code-only) composition editor, or are we going fast-forward with the new one only.


> There never was a single release of this repo, so I guess it's not any problem actually.

there was not, but we were using `#gh-pages` in bower

**Comment by warpech** [comment]
> fast-forward with the new one only

^ this :)

**Comment by warpech** [comment]
Merging

**Pull request merged**